jax/_src/nn/functions.py

```diff
@@ -275,11 +276,16 @@ def log_softmax(x: Array, axis: Optional[Union[int, Tuple[int, ...]]] = -1) -> A
     x : input array
     axis: the axis or axes along which the :code:`log_softmax` should be
       computed. Either an integer or a tuple of integers.
+    where: Elements to include in the log_softmax.
```

Comment:
`jax.numpy.max` requires `initial` to be explicitly passed when `where` is used – I think it would make sense to do the same for `log_softmax` and `softmax`. For `normalize`, it's probably not necessary because it has a natural identity.

Reply:
Sounds good. I added `initial` as an argument for `softmax` and `log_softmax`.